Dockerfile.base

```diff
@@ -1,64 +1,38 @@
 # syntax=docker/dockerfile:1
 
 # BASE docker image for music assistant container
-# Based on Debian Trixie (testing) because we need a newer version of ffmpeg (and snapcast)
-# TODO: Switch back to regular python stable debian image + manually build ffmpeg and snapcast ?
-FROM debian:trixie-slim
-
-ARG TARGETPLATFORM
+#
+# It should contain ONLY packages we want to deploy in the final image, not
+# packages needed to build. Build dependencies should be installed in the
+# "builder" image in the main Dockerfile.
 
+FROM python:3.12-alpine3.21
 
 RUN set -x \
-    && apt-get update \
-    && apt-get install -y --no-install-recommends \
+    && apk add --no-cache \
         ca-certificates \
-        curl \
-        git \
-        wget \
+        jemalloc \
         tzdata \
-        python3 \
-        python3-venv \
-        python3-pip \
-        libsox-fmt-all \
-        libsox3 \
```

Comment:
libsox should be needed for ffmpeg or is it statically linked ?

Reply:
If they were only meant for ffmpeg I guess they can be removed.

Reply:
yes, we need libsox in ffmpeg but if we switch to a statically linked version, it should be included.